### PR TITLE
Upload Files

### PR DESCRIPTION
**Pull request by vikasranatalentelgia**
Create Folder, remove Folder, List View, Image Add and Delete

### CODE REVIEW

File: src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx [hunk 1]
```diff
@@ -74,6 +75,7 @@ const UploadFileUI = ({
   uploadFileList,
   removeUploadFiles,
   GetUploadFileAndFolder,
+  getFileListFromFolderID,
```
Comment (akuks) [inline src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx:78]: `const [fileName, setfileName] = useState("");` does not seems to be used

File: src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx [hunk 2]
```diff
@@ -90,6 +92,7 @@ const UploadFileUI = ({
     previewPath: "",
     previewName: "",
   });
+  const [filename, setfilename] = useState([]);
```
Comment (akuks) [inline src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx]: Duplicated from line 84

File: src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx [hunk 3]
```diff
@@ -74,6 +75,7 @@ const UploadFileUI = ({
   uploadFileList,
   removeUploadFiles,
   GetUploadFileAndFolder,
```
Comment (akuks) [inline src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx:77]: Name is not matching with the other params. is it correct?

File: src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx [hunk 4]
```diff
@@ -277,24 +250,28 @@ const UploadFileUI = ({
...
               ) : obj.type == "folder" ? (
                 <FolderFilled className={styles.folder_icons} />
-              ) : obj.type == "text/plain" ? (
+              ) : obj.file_type == "text/plain" ? (
                 <FileTextFilled className={styles.folder_icons_fileTxt} />
-              ) : obj.type == "application/pdf" ? (
+              ) : obj.file_type == "application/pdf" ? (
                 <FilePdfFilled className={styles.folder_icons_pdf} />
               ) : (
                 <FileUnknownFilled className={styles.folder_icons} />
```
Comment (akuks) [inline src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx]: from line 256 to 27o. It is getting complicated. Move it to separate function and return from there. And please replace multiple tertiary operator with associate arrays.

File: src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx [hunk 5]
```diff
@@ -494,55 +483,59 @@ const UploadFileUI = ({
...
+                              .format("MMM DD, YYYY, h:mm:ss A")}
+                          </span>
+                        </Card>
+                      );
+                    })
+                  : ""}
+              </div>
             </div>
-          ) : (
+          ) : !openFolder ? (
```
Comment (akuks) [inline src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx]: As discussed over skype, please refactor this.

File: src/components/ComponentPages/UploadFiles/index.tsx [hunk 6]
```diff
@@ -114,11 +125,16 @@ const UploadFiles = () => {
     dragBoxHide();
     showFiles();
     uploadOptionsHide();
+    /////
```
Comment (akuks) [inline src/components/ComponentPages/UploadFiles/index.tsx]: If possible, please remove this uncommented code

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx (hunk 1 above)
Comment (SidhantUppalSyantaq) [reply src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx:78]: Done

Comment (vikasranatalentelgia) [reply src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx:78]: Done

File: src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx (hunk 2 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx]: Done

File: src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx (hunk 3 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx:77]: Yes its correct

File: src/components/ComponentPages/UploadFiles/index.tsx (hunk 6 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/UploadFiles/index.tsx]: done

File: src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx (hunk 5 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx]: done

File: src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx (hunk 4 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx]: done

**Review by akuks — APPROVED** [review]

**Pull request merged**